openmetadata-spec/src/main/resources/json/schema/metadataIngestion/databaseServiceProfilerPipeline.json

```diff
@@ -57,6 +57,12 @@
       "default": true,
       "title": "Compute Metrics"
     },
+    "useStatistics": {
```

Comment:
thinking out loud: should this be the default behavior? Thinking if we should just document this instead of letting it up to users to have it "the right way"

Reply:
Statistics requires users to configure some scheduled job to gather the statistics (running the [ANALYZE statement](https://github.com/open-metadata/OpenMetadata/pull/18433/files#diff-526b6b1c8377bfcb1753df3b0c16e70db400629ef172fcf0c7779162697a0653R111)). This is not enabled by default on trino. We can add it to the profiler best practices so that users can find this feature when optimizing the profiler.

If we enable this be default the profiler might not return any values for an out-of-the-box trino cluster.

Reply:
got it thanks